packages/core/utils/src/generateCertificate.js

```diff
@@ -63,12 +64,30 @@ export default async function generateCertificate(
     },
   ];
 
+  let altNames = [
+    {
+      type: 2, // DNS
+      value: 'localhost',
+    },
+    {
+      type: 7, // IP
+      ip: '127.0.0.1',
+    },
+  ];
+
+  if (host) {
+    altNames.push({
+      type: 2, // DNS
```

Comment:
Not entirely sure if we should check if host is an ip or domain here to update type accordingly. I think type 2 should work with both ip and domain.